src/locales/en.json

```diff
@@ -235,7 +235,7 @@
     "data_inputs_lead": "This website contains three outputs of our investigation",
     "data_inputs_calculator": "a <1>calculator</1> that you can use to calculate your COVID risk",
     "data_inputs_whitepaper": "a <1>white paper</1> that explains our estimation method",
-    "data_inputs_spreadsheet": "and a <1>spreadsheet</1> to compute your COVID risk in more detail and to track your risk over time.",
+    "data_inputs_spreadsheet": "and a <1>Risk Tracker spreadsheet</1> to compute your COVID risk in more detail and to track your risk over time.",
```

Comment:
Ah, this reads awkwardly. If Risk Tracker is a proper noun and spreadsheet is the type of thing it is, this is equivalent to writing "you should talk to a Jeremy Blanchard coach"

Reply:
I'd suggest using "risk tracking spreadsheet" or "Risk Tracker" or "Risk Tracker, a spreadsheet for computing your COVID risk in more detail and for tracking your risk over time"